application/ui/src/constants/keyboard.interface.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+export enum KeyboardEvents {
```

Comment:
On one of the previous tech talks (or on UI chat) we discussed abandoning enums over flat types. Should we revert that? wdyt?

Reply:
Yes, I was using Geti Classic’s code and forgot about it. I’ll revert it in my next commit